targets/targets.json

```diff
@@ -1472,6 +1472,23 @@
         "release_versions": ["2", "5"],
         "device_name": "STM32L433RC",
         "bootloader_supported": true
+    },
+	"WISE_1510": {
```

Comment:
no release version? will be added later?

Reply:
I'm not sure about which revision it's aiming.
Should it be current 5.7 or next release 5.8 ?

Reply:
release_versions can be 2 or 5 or both.  I assume 5 should be defined here

Reply:
oh, ok, I add it now.